containers/wordpress/wp_opt/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files
+# (the "Software"), to deal in the Software without restriction,
+# including without limitation the rights to use, copy, modify, merge,
+# publish, distribute, sublicense, and/or sell copies of the Software,
+# and to permit persons to whom the Software is furnished to do so, 
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+# OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+# OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# SPDX-License-Identifier: MIT
+ARG baseimage=wp_base
+FROM $baseimage
+LABEL authors="nitin.tekchandani@intel.com,lin.a.yang@intel.com,uttam.c.pawar@intel.com"
+
+COPY files/perf.fdata /home/${USERNAME}/oss-performance
+
+# Optimization: Set environment variable to have zend use large pages
+ENV USE_ZEND_ALLOC_HUGE_PAGES=1 
+
+# Build php 7.4.16 for bolting
+RUN sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 7 && \
+    sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 7 && \
+    cd /home/${USERNAME}/oss-performance && \
+    git clone https://github.com/php/php-src.git && \
+    cd /home/${USERNAME}/oss-performance/php-src && \
+    git checkout PHP-7.4.16 && \ 
+    ./buildconf --force && \
+    ./configure --enable-fpm --with-mysqli --with-pdo-mysql --enable-pcntl && \
+    EXTRA_CFLAGS="-g" LDFLAGS="-Wl,--emit-relocs,-znow" make -j && \
+    sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 9 && \
+    sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 9 && \
```

Comment:
Why do we also need gcc v9? Also if already have perf.fdata file do we really need gcc-v7?

Reply:
Yes. Bolting does not properly work without gcc-7. gcc-9 is default for ubuntu 20.

Reply:
We should open a task to try using gcc v9 (instead of gcc v7) with option "-fcf-protection=none" since we already have perf.fdata. This is for future PR if it works.

Reply:
Agreed